automation/check-patch.packages.el7

```diff
@@ -0,0 +1,9 @@
+yum
+git
+https://releases.hashicorp.com/vagrant/1.9.7/vagrant_1.9.7_x86_64.rpm
```

Comment:
what's the host OS you all are using?  there are fedora repo packages for vagrant and vagrant-cashier.

Reply:
CentOS 7 (As indicated by the file name).

I'd rather use CentOS as the testing base platform so I don't have to debug things every 6 months.